libs/examples/src/table/table-favorite-column-example/table-favorite-column-example.html

```diff
@@ -0,0 +1,26 @@
+<dt-table [dataSource]="dataSource" dtSort #sortable>
```

Comment:
These examples show up in our documentation as live examples and are usually copy and pasted as a guide to get started somewhere. 
Maybe you could split this example up into multiple ones, which show the different options for the `favorite column`?

Reply:
I divided in two examples and added as a reference in the `README` file